dev-tools/packaging/packages.yml

```diff
@@ -482,6 +482,8 @@ shared:
       user: '{{ .BeatName }}'
       linux_capabilities: ''
       image_name: ''
+      unpack_beats: true
```

Comment:
If we're saying this is the default behavior, do we need this option, or should we just always do this?

Reply:
I think we should make this the default and remove this option.

Reply:
Ok, removed the option and it's now unpacking by default